data/lib/compat/compat.lua

```diff
@@ -37,6 +37,11 @@ SOUTHEAST = DIRECTION_SOUTHEAST
 NORTHWEST = DIRECTION_NORTHWEST
 NORTHEAST = DIRECTION_NORTHEAST
 
+AREA_CROSS5X5 = AREA_CIRCLE5X5
```

Comment:
When @marksamman said that in #1897 I understood to put all areas in compat, and compat is not for that. So I created a special lib for them. Now I understand to explicit only the changed ones. I will revert everything then.